### ISSUE TITLE
Custom model deployment

### ISSUE THREAD

**Issue by KALALRANJITHKUMAR** [openvinotoolkit/cvat#3444]
Hello team, any idea why we are getting this error?

nuctl deploy --project-name cvat   --path serverless/custommodel/nuclio   --volume `pwd`/serverless/common:/opt/nuclio/common   --platform local
21.07.21 07:43:48.664                     nuctl (I) Deploying function {"name": ""}
21.07.21 07:43:48.664                     nuctl (I) Building {"versionInfo": "Label: 1.5.16, Git commit: ae43a6a560c2bec42d7ccfdf6e8e11a1e3cc3774, OS: linux, Arch: amd64, Go version: go1.14.3", "name": ""}
21.07.21 07:43:48.665                     nuctl (W) Failed to create a function; setting the function status {"err": "Failed create runtime", "errVerbose": "\nError - Build path is directory - runtime must be specified\n    /nuclio/pkg/processor/build/builder.go:862\n\nCall stack:\nBuild path is directory - runtime must be specified\n    /nuclio/pkg/processor/build/builder.go:862\nFailed create runtime\n    /nuclio/pkg/processor/build/builder.go:216\nFailed create runtime", "errCauses": [{"error": "Build path is directory - runtime must be specified", "errorVerbose": "\nError - Build path is directory - runtime must be specified\n    /nuclio/pkg/processor/build/builder.go:862\n\nCall stack:\nBuild path is directory - runtime must be specified\n    /nuclio/pkg/processor/build/builder.go:862\nBuild path is directory - runtime must be specified", "errorCauses": [{}]}]}

Error - Build path is directory - runtime must be specified
    /nuclio/pkg/processor/build/builder.go:862

Call stack:
Build path is directory - runtime must be specified
    /nuclio/pkg/processor/build/builder.go:862
Failed create runtime
    /nuclio/pkg/processor/build/builder.go:216
Failed to deploy function
    ...//nuclio/pkg/platform/abstract/platform.go:182

**Comment by nmanovic** [comment]
@KALALRANJITHKUMAR , could you please specify the content of the directory ' serverless/custommodel/nuclio'?

**Comment by KALALRANJITHKUMAR** [comment]
function. Yaml and main. Py

**Comment by nmanovic** [comment]
@KALALRANJITHKUMAR , can you deploy builtin models? Could you please share your custom function to reproduce the issue on our side?

**Comment by vgupta13** [comment]
@nmanovic in between, I have a doubt on how to create our own function.yaml, main.py and model_loader.py files for the custom model. Could you please help me in creating these files or share the documentation, if available?

**Comment by vgupta13** [comment]
@nmanovic in between, I have a doubt on how to create our own function.yaml, main.py and model_loader.py files for a fine tuned model on my custom dataset from the pre-trained model zoo of tensorflow (e.g., ssdmobilenet). Could you please help me in creating these files or share the documentation, if available? 

As far as I remember, in the previous version of cvat, it was possible through the use of .bin and .xml files (derived from the model optimizer) along with the label_map.json and interp.py script. But, with the architectural changes (e.g., introduction of nuclio for serverless deployment, etc), the tradional way is deprecated; hence making things difficult for me to grasp.

PS: I have gone through the serverless-tutorial instructions, but it didn't solve my problem. (Link to the tutorial: https://github.com/openvinotoolkit/cvat/blob/49e4875dad6d2b61d306078f25d05e69dab97f50/site/content/en/docs/manual/advanced/serverless-tutorial.md)

**Comment by nmanovic** [comment]
@KALALRANJITHKUMAR , we didn't get enough information from you to help. I will close the issue.
